test/lib/test-collector-validators.js

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2024 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const util = require('node:util')
+
+class CollectorValidators {
```

Comment:
I could be convinced to make this a simple object export. I started with the `class` because I assumed the validators would have been used much more thoroughly, and would have needed access to some common data. But almost all of them, except for the two replicated here, are completely unused.